setup.py

```diff
@@ -28,11 +28,15 @@ def getversion():
     maintainer_email='anthonyshaw@apache.org',
     license='MIT',
     url='https://github.com/tonybaloney/pytest-nunit',
-    description='A pytest plugin for generating Nunit3 test result XML output',
+    description='A pytest plugin for generating NUnit3 test result XML output',
     long_description=read('README.rst'),
     packages=find_packages(exclude=('test*',)),
     python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
-    install_requires=['pytest>=3.5.0', 'enum34', 'attrs'],
+    install_requires=['pytest>=3.5.0', 'attrs'],
+    extras_require={
+        ':python_version=="2.7"': ['enum34>=1.1.6'],
+        ':python_version=="3.4"': ['enum34>=1.1.6'],
```

Comment:
you don't need `enum34` on python3.4 -- that's when it got introduced (and it won't be earliy enough on `sys.path` to matter any way since `site-packages` comes after stdlib)